src/pthread-main.js

```diff
@@ -69,6 +74,8 @@ this.onmessage = function(e) {
     Runtime.establishStackSpace(e.data.stackBase, e.data.stackBase + e.data.stackSize);
     var result = 0;
 
+    PThread.receiveObjectTransfer(e.data);
```

Comment:
When the 'run' command is posted, we send the GL canvas objects there as well. We could send a separate 'objectTransfer' message first, but I think it's better to just send one message.